packages/form/src/ux-field.ts

```diff
@@ -0,0 +1,21 @@
+import { customElement, bindable } from 'aurelia-templating';
+import { inject } from 'aurelia-dependency-injection';
+
+@inject(Element)
+@customElement('ux-field')
+
+export class UxField {
+  @bindable public label: string;
+
+  constructor(private element: Element) { }
+
+  public attached() {
+    if (this.label && !this.element.querySelector('label')) {
+      const newLabel = document.createElement('label');
+
+      newLabel.textContent = this.label;
```

Comment:
If someone databinds the label property, the generated/located label will need to update in response to that.